config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-0-3.yaml

```diff
@@ -51,7 +51,7 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
 - name: periodic-cluster-api-verify-book-links-release-0-3
-  interval: 24h
+  interval: 12h
```

Comment:
That would run actually more often now. I missed this on the issue. Can you keep it at 24h please?